python/pfs/drp/stella/datamodel/pfsFiberNorms.py

```diff
@@ -1,77 +0,0 @@
-from typing import Optional, Tuple, TYPE_CHECKING
-import warnings
-
-import numpy as np
-
-import pfs.datamodel
-
-from ..utils.math import robustRms
-
-if TYPE_CHECKING:
-    import matplotlib
-
-__all__ = ("PfsFiberNorms",)
-
-
-class PfsFiberNorms(pfs.datamodel.PfsFiberNorms):
```

Comment:
I don't think you should remove this file completely (besides other concerns, it will break the pipeline). I think this should stay, along with the `plot` method. If you remove this, I won't be able to call `fiberNorms.plot()` in Jupyter.

Reply:
What is the logic for what goes in the `drp_stella` override of the datamodel vs what goes into the datamodel class directly?  Should we never be importing directly from `pfs.datamodel`?

Reply:
The datamodel is concerned solely with I/O. The layer over the top in drp_stella provides algorithmic content.